src/Unleash/DefaultUnleash.cs

```diff
@@ -58,7 +66,13 @@ public DefaultUnleash(UnleashSettings settings, bool overrideDefaultStrategies,
 
             services = new UnleashServices(settings, strategyMap);
 
-            Logger.Info($"UNLEASH: Unleash is initialized and configured with: {settings}");
+            Logger.Info($"UNLEASH: Unleash instance number { currentInstanceNo } is initialized and configured with: {settings}");
+
+            if (currentInstanceNo >= WarnOnInstanceCount)
+            {
+                Logger.Warn($"UNLEASH: Unleash instance count for this process is now {currentInstanceNo}.");
+                Logger.Warn("Ideally you should only need 1 instance of Unleash per app/process, we recommend setting up Unleash as a singleton.");
```

Comment:
Idle thoughts but I do... uh... I do strongly recommend this haha 

```suggestion
                Logger.Warn("Ideally you should only need 1 instance of Unleash per app/process, we strongly recommend setting up Unleash as a singleton.");
```